stytch/core/test/test_models.py

```diff
@@ -108,3 +109,22 @@ def test_stytcherror(self) -> None:
             mock_details = create_autospec(models.StytchErrorDetails)
             str(models.StytchError(mock_details))
             mock_details.__str__.assert_called_once()
+
+    def test_stytcherror_fields(self) -> None:
+        resp = '''{
+          "status_code": 418,
+          "request_id": "request-id-test-fea11c44-5514-4aac-a76b-3ca685e3443a",
+          "error_type": "is_a_teapot",
+          "error_message": "I'm a teapot!",
+          "error_url": "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418"
+        }'''
```

Comment:
[pebble] Honestly, you could probably just store this as a dictionary since we're not especially interested in testing the capabilities of the `json.loads` function call.

Reply:
Ah good point. I was just being lazy about copying the data out of my shell 😅 

319f7a0c8e5a76e9f3175714f2a71a4b50d4a67e